decidim-home/app/views/layouts/decidim/_wrapper.html.erb

```diff
@@ -102,11 +102,12 @@ end
           </nav>
         </div>
 
-        <main>
+        <main>  
           <%= display_flash_messages %>
           <%= yield %>
         </main>
       </div><!-- /.footer-separator -->
+      <%= render partial: "layouts/decidim/main_feder_footer" %>
```

Comment:
Instead of overriding, it will be good to use the `deface` gem like it was done in decidim-barcelona: https://github.com/AjuntamentdeBarcelona/decidim-barcelona/pull/300

Reply:
Thanks for the alternative solution, applied on next commit